src/host/ncp_spinel.cpp

```diff
@@ -750,6 +769,15 @@ void NcpSpinel::HandleValueRemoved(spinel_prop_key_t aKey, const uint8_t *aBuffe
         break;
     }
 #endif // OTBR_ENABLE_SRP_ADVERTISING_PROXY
+    case SPINEL_PROP_BACKBONE_ROUTER_MULTICAST_LISTENER:
+    {
+        const otIp6Address *addr;
+
+        VerifyOrExit(decoder.ReadIp6Address(addr) == OT_ERROR_NONE, error = OTBR_ERROR_PARSE);
```

Comment:
Is it calling this one? https://github.com/openthread/openthread/blob/819938d05dae7393d19f2c02a8e5f0aa99900efb/src/lib/spinel/spinel_decoder.hpp#L245

Reply:
It's calling this one: https://github.com/openthread/openthread/blob/819938d05dae7393d19f2c02a8e5f0aa99900efb/src/lib/spinel/spinel_decoder.hpp#L261

The two are identical.